### ISSUE TITLE
[BUG] module sql_cacher has encoding issue in parameter column statement

### ISSUE THREAD

**Issue by netaskd** [OpenSIPS/opensips#1900]
**OpenSIPS version you are running**
```
opensips -V
version: opensips 3.0.1 (x86_64/linux)
flags: STATS: On, DISABLE_NAGLE, USE_MCAST, SHM_MMAP, PKG_MALLOC, Q_MALLOC, F_MALLOC, HP_MALLOC, DBG_MALLOC, FAST_LOCK-ADAPTIVE_WAIT
ADAPTIVE_WAIT_LOOPS=1024, MAX_RECV_BUFFER_SIZE 262144, MAX_LISTEN 16, MAX_URI_SIZE 1024, BUF_SIZE 65535
poll method support: poll, epoll, sigio_rt, select.
main.c compiled on 02:40:49 Oct  2 2019 with gcc 4.8.5
```

**Describe the bug**
config contains:
```
loadmodule "sql_cacher.so"
modparam("sql_cacher", "sql_fetch_nr_rows", 1000)
modparam("sql_cacher", "full_caching_expire", 86400) # 24h
modparam("sql_cacher", "reload_interval", 60) # seconds before the cache expires
modparam("sql_cacher", "pvar_delimiter", ":")
modparam("sql_cacher", "columns_delimiter", ",")
modparam("sql_cacher", "cache_table",
"id=by_id
db_url=mysql://opensips:XXXXXX@127.0.0.1:3306/opensips_3_0
cachedb_url=local://
table=dispatcher
key=id
columns=destination,state,attrs,id")
```
after reload I get the test error:
```
Nov 21 11:01:21 [187] CRITICAL:db_mysql:wrapper_single_mysql_real_query: driver error (1064): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '? from dispatcher where id='sql_cacher_test_query_key'' at line 1
Nov 21 11:01:21 [187] ERROR:core:db_do_query: error while submitting query - [select destination,state,attrs,idX0\ufffd? from dispatcher where id='sql_cacher_test_query_key']
---------------------------------------------------------------------------------------------------------------^
Nov 21 11:01:21 [187] ERROR:sql_cacher:db_init_test_conn: Failure to issuse test query to SQL DB: mysql://opensips:XXXXXX@127.0.0.1:3306/opensips_3_0
```
so, symbol `X0\ufffd?` (some times `3\ufffd?`) added in the latest param. Looks like encoding problem, but I tried replace it from scratch, the same effect.

**OS/environment information**
 - Operating System: CentOS Linux release 7.6.1810 (Core)
 - OpenSIPS installation: 3.0.1 release rpm repo

**Comment by razvancrainea** [comment]
@netaskd could you confirm this only happens after running `reload_routes` mi command?

**Comment by netaskd** [comment]
@razvancrainea nope. It happens when opensips is killed and started again.

**Comment by netaskd** [comment]
@razvancrainea, it's really strange, but I can't reproduce it on `Centos 7.7.1908` and `Opensips 3.0.1.20191205.fc728f0c5-1.el7`. Config the same.

**Comment by razvancrainea** [comment]
I've tried to replicate this myself and didn't replicate. I've also audited the code, and it looks just fine. @rvlad-patrascu has a bit more experience with that code - did you manage to take a look at this issue?

**Comment by netaskd** [comment]
I've caught it again on a prod, but can't re-produce it on a stage. Code the same, version the same (it's just the next directory on the same server with different binding ports)
So, I made a workaround for this:
```
modparam("sql_cacher", "cache_table",
"id=by_id
db_url=mysql://opensips:XXXXXX@127.0.0.1:3306/opensips_3_0
cachedb_url=local://
table=dispatcher
key=id
columns=destination,state,attrs,`id`")
```
I've wripped it to \`id\` symbols because it hadn't have passing a test when opensips restarted. Actually "id" is not used in logic, it was added only for workaround.  I need attrs and with this WA it works normally.
